Sources/eppo/EppoClient.swift

```diff
@@ -45,14 +45,59 @@ public class EppoClient {
         self.flagConfigs = try JSONDecoder().decode(FlagConfigJSON.self, from: urlData);
     }
 
-    public func getAssignment(_ subjectKey: String, _ flagKey: String) throws -> String? {
+    public func getBoolAssignment(
+        _ subjectKey: String,
+        _ flagKey: String) throws -> Bool?
+    {
+        return try getAssignment(subjectKey, flagKey, SubjectAttributes())?.boolValue()
+    }
+    
+    public func getBoolAssignment(
+        _ subjectKey: String,
+        _ flagKey: String,
+        _ subjectAttributes: SubjectAttributes) throws -> Bool?
+    {
+        return try getAssignment(subjectKey, flagKey, subjectAttributes)?.boolValue()
+    }
```

Comment:
I think we can collapse these into one function declaration [leveraging default parameter values](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions#Default-Parameter-Values).

```swift
public func getBoolAssignment(
    _ subjectKey: String,
    _ flagKey: String,
    _ subjectAttributes: SubjectAttributes = SubjectAttributes()) throws -> Bool?
{
    return try getAssignment(subjectKey, flagKey, subjectAttributes)?.boolValue()
}
```

Same for the other types below